google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -81,7 +81,7 @@ public static Builder newBuilder() {
     private int minSessions;
     private int maxSessions = DEFAULT_MAX_SESSIONS;
     private int maxIdleSessions;
-    private float writeSessionsFraction = 0.2f;
+    private float writeSessionsFraction = 0.f;
```

Comment:
I don't think we want to change this default, since maintaining some write sessions does in the general case improve performance.

Reply:
That's correct. But I think anyone who cares about performance improvements provided by this parameter would anyways need to tune this according to there read/write ratio. 0.2 is pretty arbitrary.
OTOH setting this to >0 does lead to a bad experience in the case where BeginTransaction is guaranteed to fail such as permission denied.